### PR TITLE
Add benchmarking for parachain runtime configuration pallet

### PR DESCRIPTION
**Pull request by KiChjang**
Part of #3850.

### CODE REVIEW

File: runtime/parachains/src/configuration.rs [hunk 1]
```diff
@@ -613,7 +663,7 @@ pub mod pallet {
 		}
 
 		/// Sets the soft limit for the phase of dispatching dispatchable upward messages.
-		#[pallet::weight((1_000, DispatchClass::Operational))]
+		#[pallet::weight(T::WeightInfo::set_ump_service_total_weight())]
```
Comment (shawntabrizi) [inline runtime/parachains/src/configuration.rs:666]: practically speaking, we could probably make one benchmark for all extrinsics which are simply `new: u32`, which will simplify maintenance a lot.

File: runtime/parachains/src/configuration.rs [hunk 2]
```diff
@@ -665,7 +715,7 @@ pub mod pallet {
 
 		/// Sets the amount of funds that the recipient should provide for accepting opening an HRMP
 		/// channel.
-		#[pallet::weight((1_000, DispatchClass::Operational))]
+		#[pallet::weight(T::WeightInfo::set_hrmp_recipient_deposit())]
```
Comment (shawntabrizi) [inline runtime/parachains/src/configuration.rs:718]: and then a single benchmark for `new: Balance`... etc...

**Review by shawntabrizi — APPROVED** [review]
lgtm

**Review by adoerr — APPROVED** [review]

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**